java/developer-guide/working-with-outlook-items/working-with-recurrences/_index.md

```diff
@@ -0,0 +1,527 @@
+---
+title: Working with Recurrences
+type: docs
+weight: 140
+url: /java/working-with-recurrences/
+---
+
+
+## **Working with Daily Recurrences**
+Aspose.Email supports creation of daily recurrences using MapiCalendarDailyRecurrencePattern. Three different Mapi calendar recurrence end types can be used including EndAfterNOccurrences, EndAfterDate and NeverEnd. This section demonstrates the creation of different daily recurrence patterns.
+### **Daily Recurrences: EndAfterNOccurrence Recurrance Type**
+In this type of recurrence, number of recurrences is to be set along with other information as follows:
+
+1. Set start, end and due date.
+1. Create a MapiTask.
+1. Set task state to NotAssigned.
+1. Create the daily recurrence object by setting the properties like PatternType, Period, WeekStartDay, EndType and OccurenceCount.
```

Comment:
`1. Create the daily recurrence object by setting the properties like *PatternType*, *Period*, *WeekStartDay*, *EndType* and *OccurenceCount*.`